products/confluence.md

```diff
@@ -111,7 +111,7 @@ Confluence is available both as SaaS and on-premises offer, with three editions:
 - Confluence Cloud: Cloud edition, available through [Atlassian Cloud](https://www.atlassian.com/licensing/cloud).
 - Confluence Server: Self-hosted edition.
 - Confluence Data Center: Self-hosted edition, targeted to enterprises (see [Confluence Server and Data
-  Center feature comparison](https://confluence.atlassian.com/doc/confluence-server-and-data-center-feature-comparison-953652032.html)
+  Center feature comparison](https://confluence.atlassian.com/display/DOC/Confluence+Server+and+Data+Center+feature+comparison)
```

Comment:
```suggestion
  Center feature comparison](https://confluence.atlassian.com/conf85/confluence-server-and-data-center-feature-comparison-1283361452.html)
```

https://confluence.atlassian.com/display/DOC/Confluence+Server+and+Data+Center+feature+comparison is also a broken page.